$OpenBSD: patch-docs_conf_py,v 1.1 2017/11/02 09:28:26 sebastia Exp $

For python 2.7, the get_distribution fails with:
raise DistributionNotFound(req, requirers)
DistributionNotFound: The 'Flask-WTF' distribution was not found and is
required by the application

hardcode the version and release as prior versions of flask-wtf did.
see: https://github.com/lepture/flask-wtf/issues/310

Index: docs/conf.py
--- docs/conf.py.orig
+++ docs/conf.py
@@ -49,11 +49,11 @@ author = 'Dan Jacob'
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-dist = get_distribution('Flask-WTF')
+#dist = get_distribution('Flask-WTF')
 # The short X.Y version.
-version = '.'.join(dist.version.split('.', 2)[:2])
+version = '${MAIN_VERSION}'
 # The full version, including alpha/beta/rc tags.
-release = dist.version
+release = '${MODPY_EGG_VERSION}'
 
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
